.github/CODEOWNERS

```diff
@@ -927,10 +927,10 @@
 # AzureSdkOwners:                                                  @ArthurMa1978
 
 # PRLabel: %ARM
-/sdk/resources/Azure.ResourceManager.*/                            @Azure/arm-sdk-owners
+/sdk/resources/Azure.ResourceManager.*/                            @arm-sdk-owners
```

Comment:
This is not a known GitHub user.    If this is a team, the format is incorrect.  The organization under which the team was created needs to be used as a prefix, as  it was in the previous iteration.

Reply:
I just added arm-sdk-owners as a child of azure-sdk-write but a maintainer of that team will have to except that change.

Reply:
@weshaggard can I know by when would we expect to know the approval from maintainer?

Reply:
You will have to ask a maintainer of that team to accept the parent update. @cemheren @majastrz or @anthony-c-martin to accept the update to re-parent arm-sdk-owners to azure-sdk-write.

Reply:
I just approved the change.

Reply:
Never mind. I only approved the PR. I haven't seen the request to reparent the group.

Reply:
@majastrz if you go to the team https://github.com/orgs/Azure/teams/arm-sdk-owners and look at the settings it should show a request to parent to azure-sdk-write.

Reply:
Finally found the approval email and approved it. The team should be parented correctly now:
![image](https://github.com/Azure/azure-sdk-for-net/assets/22460039/c7d2a36e-cad9-4db7-8a83-2b3a36e10f5c)